src/directives/message_media.ts

```diff
@@ -156,7 +156,9 @@ export default [
                         this.wasInView = inView;
 
                         if (!inView) {
-                            timeoutService.cancel(loadingThumbnailTimeout);
+                            if (loadingThumbnailTimeout != null) {
```

Comment:
`!==`

Also, where is this set to `null`?

Reply:
It's the default.